gravitee-apim-console-webui/src/management/settings/groups/group/group.component.html

```diff
@@ -162,69 +162,65 @@ <h2>{{ mode === 'new' ? 'Create group' : group.name }}</h2>
     <mat-card class="group__form__card">
       <mat-card-header>
         <mat-card-subtitle>Dependents</mat-card-subtitle>
-
-        @if (canAddMembers) {
+        <button
+          class="group__form__card__actions"
+          mat-raised-button
+          color="primary"
+          [disabled]="!canAddMembers"
+          [matMenuTriggerFor]="addMembersMenu"
+          matTooltip="Search and invite users to the group"
+        >
+          <mat-icon>add</mat-icon>
```

Comment:
can you use the svg icon from ui-particles please ?

```suggestion
          <mat-icon svgIcon="gio:plus"></mat-icon>
```

Reply:
Done.